src/js/components/SkipLinks.js

```diff
@@ -0,0 +1,75 @@
+// (C) Copyright 2014-2015 Hewlett-Packard Development Company, L.P.
+
+var React = require('react');
+var Layer = require('./Layer');
+var Menu = require('./Menu');
+var DOM = require('../utils/DOM');
+var IntlMixin = require('../mixins/GrommetIntlMixin');
+
+var SkipLinks = React.createClass({
+
+  mixins: [IntlMixin],
+
+  componentDidMount: function () {
+    var anchorElements = document.querySelectorAll('[data-skip-label]');
```

Comment:
Ran some more tests. This method for getting a document to use should work. What you may not see is the bound event listener in the DOM the same way (such as in chrome devtools). Things should still work.